python/quartz/_cython/CCore.pxd

```diff
@@ -0,0 +1,33 @@
+cdef extern from "context/context.h" namespace "quartz":
```

Comment:
ditto (and also `python/quartz/_cython/core.pyx` and `python/setup.py`)

Reply:
We should keep these three files because they are used to wrap the cpp library into python interface.